lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -303,13 +303,28 @@ def whitespace_after?(range, length = 1)
 
         def move_comment_before_block(corrector, comment, block_node, closing_brace)
           range = block_node.chained? ? end_of_chain(block_node.parent).source_range : closing_brace
+
+          # it is possible that there is code between the block and the comment
+          # which needs to be preserved and trimmed
```

Comment:
That's just my two cents. Can you tweak it?

```suggestion
          # It is possible that there is code between the block and the comment
          # which needs to be preserved and trimmed.
```

Reply:
Sure happy to. Updated!